### ISSUE TITLE
is there an equivalent of `:GoErrCheck` in ray-x/go.nvim like in fatih/vim-go ?

### ISSUE THREAD

**Issue by stephane-archer** [ray-x/go.nvim#404]

**Comment by ray-x** [comment]
Yes, Please use `GoLint` instead.  and make sure you installed `golaingci-lint` and enabled `errcheck` for `golaingci-lint`

**Comment by stephane-archer** [comment]
How do I enable `errcheck`  for `golaingci-lint`? in my lua config?

**Comment by ray-x** [comment]
you may need a golangci-lint config file: https://golangci-lint.run/usage/configuration#config-file

**Comment by stephane-archer** [comment]
Thank you
Why did you choose to not enable all available linters?
